README.md

```diff
@@ -101,21 +101,13 @@ key is in this repo under [deb.asc](/deb.asc).
 To install eza from this repo use:
 
 ```bash
-wget -qO- https://raw.githubusercontent.com/eza-community/eza/main/deb.asc | sudo tee /etc/apt/trusted.gpg.d/gierens.asc
-echo "deb http://deb.gierens.de stable main" | sudo tee /etc/apt/sources.list.d/gierens.list
+sudo mkdir /etc/apt/keyrings
```

Comment:
Please make this a `sudo mkdir -p /etc/apt/keyrings` so that is doesn't fail when the directory already exists.